pybossa/view/account.py

```diff
@@ -499,10 +500,6 @@ def create_account(user_data, project_slugs=None, ldap_disabled=True, auto_creat
                      password=user_data['password'])
     msg = generate_invitation_email_for_new_user(user=user_info, project_slugs=project_slugs)
     mail_queue.enqueue(send_mail, msg)
```

Comment:
As generate_bsso_account_notification is not in use, please remove it from [here](https://github.com/bloomberg/pybossa/pull/508/files#diff-1a25756214d69e9b8795014a754ad87cR46) and its definition from util.py.